bin/p2-dsctl/main.go

```diff
@@ -22,18 +22,19 @@ import (
 )
 
 const (
-	CmdCreate  = "create"
-	CmdGet     = "get"
-	CmdList    = "list"
-	CmdEnable  = "enable"
-	CmdDisable = "disable"
-	CmdDelete  = "delete"
-	CmdUpdate  = "update"
+	CmdCreate           = "create"
+	CmdGet              = "get"
+	CmdList             = "list"
+	CmdEnable           = "enable"
+	CmdDisable          = "disable"
+	CmdDelete           = "delete"
+	CmdUpdate           = "update"
+	CmdOverrideSelector = "overrideSelector"
```

Comment:
Unix commands typically use dashes to join words in flags/commands. So this would be `override-selector`.

But I don't think calling this an "override" is a good descriptor of what this command does. "Override" usually means something like: "Ignore that saved value, I'll supply the value to use right now." That isn't what's happening here. This command sets the DS's node selector. I think `set-selector` would be easier to understand.